### PR TITLE
Minor Spelling Mistake fixes

### PR DESCRIPTION
**Pull request by Geevies**
I only fixed spelling mistakes that get presented to the user, I may have missed some, especially those on the !help command. This is just stage one, I suppose.

### CODE REVIEW

File: EventBot/Modules/EventModule.cs [hunk 1]
```diff
@@ -288,17 +288,17 @@ public EventManagementModule(EventManagementService events, DatabaseService data
             [Command()]
             [Summary("Get info about event.")]
             public async Task EventInfo(
-            [Summary("Event about witch info is wanted.")] Event @event = null)
+            [Summary("Event  ID of event you wish to know more of.")] Event @event = null)
```
Comment (Karolis2011) [inline EventBot/Modules/EventModule.cs]:
```suggestion
            [Summary("Event ID of event you wish to know more of.")] Event @event = null)
```

File: EventBot/Modules/EventModule.cs [hunk 2]
```diff
@@ -316,12 +316,12 @@ public EventManagementModule(EventManagementService events, DatabaseService data
             }
             [Priority(1)]
             [Command("role")]
-            [Summary("Gets role info.")]
+            [Summary("Gets info about a role.")]
             public async Task EventRoleInfo(
-            [Summary("Role about witch info is wanted.")] EventRole eventRole)
+            [Summary("Role you wish to have more info about.")] EventRole eventRole)
```
Comment (Karolis2011) [inline EventBot/Modules/EventModule.cs]:
```suggestion
            [Summary("Role you wish to have info about.")] EventRole eventRole)
```

File: EventBot/Modules/EventModule.cs [hunk 3]
```diff
@@ -339,14 +339,14 @@ public EventManagementModule(EventManagementService events, DatabaseService data
             }
 
             [Command("open")]
-            [Summary("Open registration for event here.")]
+            [Summary("Open registration for event here. Warning: Ensure everything is correct before doing this.")]
```
Comment (Karolis2011) [inline EventBot/Modules/EventModule.cs]: Don't. everything is correctable. You can always close event and add new roles.
```suggestion
            [Summary("Open registration for event here.")]
```

File: EventBot/Modules/EventModule.cs [hunk 4]
```diff
@@ -387,7 +387,7 @@ public EventManagementModule(EventManagementService events, DatabaseService data
                 @event.MessageId = 0;
                 @event.MessageChannelId = 0;
                 await _database.SaveChangesAsync();
-                await ReplyAsync($"Event `{@event.Id}` registration has been closed, it's registration message will now be normal message.");
+                await ReplyAsync($"Event `{@event.Id}`'s registration has been closed, its registration message will now be normal message.");
```
Comment (Karolis2011) [inline EventBot/Modules/EventModule.cs]:
```suggestion
                await ReplyAsync($"Event's `{@event.Id}` registration has been closed, its registration message will now be normal message.");
```

File: EventBot/Modules/EventModule.cs [hunk 5]
```diff
@@ -398,14 +398,14 @@ public EventManagementModule(EventManagementService events, DatabaseService data
...
-                    throw new Exception("No events were found for this guild.");
+                    throw new Exception("No events were found for this discord server.");
                 if (!@event.Active)
                     throw new Exception("This event is already finalized.");
 
                 @event.Active = false;
                 await _database.SaveChangesAsync();
                 
-                await ReplyAsync($"Event `{@event.Id}` has been finilized. Removing participant roles...");
+                await ReplyAsync($"Event `{@event.Id}`'s has been finalized. Removing participant roles..");
```
Comment (Karolis2011) [inline EventBot/Modules/EventModule.cs]:
```suggestion
                await ReplyAsync($"Event `{@event.Id}` has been finalized. Removing participant roles..");
```

File: EventBot/Modules/EventModule.cs [hunk 6]
```diff
@@ -421,15 +421,15 @@ public async Task EventArchive()
             {
                 var guildEvents = _database.Events.Where(e => e.GuildId == Context.Guild.Id).OrderBy(e => e.Opened).ToList();
                 if (guildEvents.Count() == 0)
-                    throw new Exception("There are no events that roon on this server.");
+                    throw new Exception("No events have been run on this server.");
```
Comment (Karolis2011) [inline EventBot/Modules/EventModule.cs]:
```suggestion
                    throw new Exception("There are no events, that have been ran on this server.");
```

File: EventBot/Modules/EventModule.cs [hunk 7]
```diff
@@ -478,24 +478,24 @@ public async Task EventArchive()
             [Command("participant remove")]
             [Summary("Remove participant from event role.")]
             public async Task EventParticipantRemove(
-            [Summary("User that is participanting id or mention")] IUser user,
-            [Summary("Event to romove participant from")] Event @event = null)
+            [Summary("User that is participanting's ID or discord mention.")] IUser user,
```
Comment (Karolis2011) [inline EventBot/Modules/EventModule.cs]:
```suggestion
            [Summary("User that is participanting ID or discord mention.")] IUser user,
```

File: EventBot/Modules/EventModule.cs [hunk 8]
```diff
@@ -207,71 +207,71 @@ public EventManagementModule(EventManagementService events, DatabaseService data
...
+            [Summary("Updates the role's description.")]
             public async Task UpdateEventRoleDescription(
-            [Summary("Role witch to update.")] EventRole eventRole,
-            [Summary("New description for role.")][Remainder] string description)
+            [Summary("Which role to update.")] EventRole eventRole,
+            [Summary("New description for the role.")][Remainder] string description)
             {
                 if (eventRole == null)
-                    throw new Exception("Please provide correct role.");
+                    throw new Exception("Please provide the correct role info, this one does not exist.");
```
[the remark this answers is not in the document]
Comment (Geevies) [reply EventBot/Modules/EventModule.cs]: shall I change it to "correct role ID" then?

[the remark this answers is not in the document]
Comment (Karolis2011) [reply EventBot/Modules/EventModule.cs]: Yeah, most likely

File: EventBot/Modules/EventModule.cs [hunk 9]
```diff
@@ -207,71 +207,71 @@ public EventManagementModule(EventManagementService events, DatabaseService data
...
```
Comment (Geevies) [inline EventBot/Modules/EventModule.cs]:
```suggestion
                    throw new Exception("Please provide the correct role ID, this one does not exist.");
```

File: EventBot/Modules/EventModule.cs (hunk 6 above)
Comment (Geevies) [inline EventBot/Modules/EventModule.cs]:
```suggestion
                    throw new Exception("There are no events that have been run on this server.");
```

File: EventBot/Modules/EventModule.cs (hunk 7 above)
Comment (Geevies) [inline EventBot/Modules/EventModule.cs]:
```suggestion
            [Summary("User that is participating's ID or discord mention.")] IUser user,
```

Comment (Geevies) [reply EventBot/Modules/EventModule.cs]: Participating's is a possessive form. (User that is participating)'s ID.

File: EventBot/Modules/EventModule.cs (hunk 6 above)
Comment (Karolis2011) [reply EventBot/Modules/EventModule.cs]: `that was ran` sounds better. and gramaticly seems more correct.

**Pull request merged**